pkg/compactor/compactor_metrics.go

```diff
@@ -174,6 +175,10 @@ func newCompactorMetricsWithLabels(reg prometheus.Registerer, commonLabels []str
 		Name: "cortex_compactor_group_partition_count",
 		Help: "Number of partitions for each compaction group.",
 	}, compactionLabels)
+	m.compactionsNotPlanned = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+		Name: "cortex_compactor_group_compactions_not_planned",
+		Help: "Total number of group compaction not planned due to non-critical error (ie. group is currently visited by other compactor).",
```

Comment:
This metric is a bit confusing to me. Non-critical error... Is it recommended for users to alarm on this metric?

Reply:
Updated description and use this metric for not planned case due to error